hackclub.com.yaml

```diff
@@ -2069,3 +2069,12 @@ zwp:
   - ttl: 1
     type: A
     value: 172.99.233.117
+ morncast:
+  - ttl: 1
+    type: CNAME
+    value: hackathon-coral.vercel.app.
+  _vercel:
```

Comment:
Hey! Please add these to Vercel record at the top.

Reply:
fixed it.